lib/modules/platform/bitbucket-server/index.spec.ts

```diff
@@ -1694,16 +1694,22 @@ describe('modules/platform/bitbucket-server/index', () => {
             .put(
               `${urlPath}/rest/api/1.0/projects/SOME/repos/repo/pull-requests/5`,
             )
-            .reply(200);
+            .reply(200, {
+              ...prMock(url, 'SOME', 'repo'),
+              toRef: {
+                id: 'refs/heads/new_base',
+                displayId: 'new_base',
+                latestCommit: '0d9c7726c3d628b7e28af234595cfd20febdbf8e',
+              },
+            });
 
-          await expect(
-            bitbucket.updatePr({
-              number: 5,
-              prTitle: 'title',
-              prBody: 'body',
-              targetBranch: 'new_base',
-            }),
-          ).toResolve();
+          await bitbucket.updatePr({
+            number: 5,
+            prTitle: 'title',
+            prBody: 'body',
+            targetBranch: 'new_base',
+          });
+          expect(1).toBe(1);
```

Comment:
why this change? I would prefer the previous one

Reply:
Sorry, got left :)